full-service/src/test_utils.rs

```diff
@@ -378,11 +378,37 @@ pub fn create_test_txo_for_recipient(
     recipient_subaddress_index: u64,
     amount: Amount,
     rng: &mut StdRng,
+) -> (TxOut, KeyImage) {
+    return create_test_txo_for_recipient_with_memo(
+        recipient_account_key,
+        recipient_subaddress_index,
+        amount,
+        rng,
+        TransactionMemo::Empty,
+    );
```

Comment:
Optional/Equivalent: Omit `return`
```suggestion
    create_test_txo_for_recipient_with_memo(
        recipient_account_key,
        recipient_subaddress_index,
        amount,
        rng,
        TransactionMemo::Empty,
    )
```